sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/OperationState.java

```diff
@@ -31,7 +31,7 @@
   CANCELED(TOperationState.CANCELED_STATE, true),
   CLOSED(TOperationState.CLOSED_STATE, true),
   ERROR(TOperationState.ERROR_STATE, true),
-  UNKNOWN(TOperationState.UKNOWN_STATE, false),
+  UNKNOWN(TOperationState.UNKNOWN_STATE, false),
```

Comment:
Apparently this is a downstream for this: https://github.com/apache/hive/search?q=UKNOWN_STATE, so I'll need to change that first.

I'll drop this shortly. If you have any advice for steps I should take w/ that project, I'm open to advice. Otherwise, I'll do something similar to what I did here (it took a number of days to produce this initial PR).

Reply:
apache/hive#1674 https://github.com/apache/hive/pull/1674/commits/40b81969b47d310363f14fb751e7fb7db7bc10d3

Reply:
Yeah I just wouldn't touch the Hive code here for now. Anything non-trivial can be dealt with later.

Reply:
Yeah, in the process of dropping this (currently reviewing the build failures)